File: C++/1064.cpp
#include <iostream>
#include <iostream>
#include <string>
#include <limits>
#include <cmath>
#include <iomanip>

using namespace std;

int main()
{

    double n;
    int total = 0;
    double soma = 0;
    for (int i = 0; i < 6; i++)
    {
        cin >> n;
        if (n > 0)
        {
            total++;
            soma += n;
        }
    }

    soma = soma / total;

    cout << total << " valores positivos\n";
    cout << fixed << setprecision(1) << soma << "\n";
}

File: C#/1064.cs
using System;

class URI
{

    static void Main(string[] args)
    {
        int n, positivo = 0;
        double numero;
        double soma = 0;

        for (n = 0; n < 6; n++)
        {
            numero = Convert.ToDouble(Console.ReadLine());

            if (numero > 0)
            {
                positivo++;
                soma += numero;
            }
        }
        soma = soma / positivo;
        Console.WriteLine(positivo + " valores positivos");
        Console.WriteLine("{0:0.0}", soma);

    }
}